From f93c29b8471bee953ce4cbf17356dda474b90aa8 Mon Sep 17 00:00:00 2001
From: WingMan Kwok <w-kwok2@ti.com>
Date: Thu, 14 Dec 2017 18:51:11 -0500
Subject: [PATCH 128/140] net: ti: iep: rename function iep_enable to
 iep_ptp_feature_enable

Renames function iep_enable to iep_ptp_feature_enable. The new
name better reflects the purpose of the function. No functionality
change.

Signed-off-by: WingMan Kwok <w-kwok2@ti.com>
---
 drivers/net/ethernet/ti/iep.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/ethernet/ti/iep.c b/drivers/net/ethernet/ti/iep.c
index 6e57754..8fe8423 100644
--- a/drivers/net/ethernet/ti/iep.c
+++ b/drivers/net/ethernet/ti/iep.c
@@ -279,8 +279,8 @@ static int iep_pps_init(struct iep *iep)
 	return 0;
 }
 
-static int iep_enable(struct ptp_clock_info *ptp,
-		      struct ptp_clock_request *rq, int on)
+static int iep_ptp_feature_enable(struct ptp_clock_info *ptp,
+				  struct ptp_clock_request *rq, int on)
 {
 	struct iep *iep = container_of(ptp, struct iep, info);
 
@@ -421,7 +421,7 @@ static long iep_overflow_check(struct ptp_clock_info *ptp)
 	.adjtime	= iep_adjtime,
 	.gettime64	= iep_gettime,
 	.settime64	= iep_settime,
-	.enable		= iep_enable,
+	.enable		= iep_ptp_feature_enable,
 	.do_aux_work	= iep_overflow_check,
 };
 
-- 
1.9.1

